README.md

```diff
@@ -13,32 +13,48 @@ For a longer description and available features and options, see the
 
 # Installation
 
-Installing mkosi is easy, as it has no runtime Python dependencies (you will
-need all the tools to format filesystems and bootstrap the distribution
-appropriate for your image, though).
+You can install mkosi from your distribution with its package manager or the
+development version from git. The distribution packages are the preferred way to
+install mkosi.
 
-If you just want the main branch you can run
+The development version of mkosi might require tools from the systemd main
+branch, see the [`action.yaml`](action.yaml) for what we currently use in CI.
+
+## Alternative Installation methods
+
+If you just want to give the development version of mkosi a quick spin you can run
+```shell
+pipx install git+https://github.com/systemd/mkosi.git
+```
+which will transparently install mkosi into a Python virtual environment and a mkosi
+binary to `~/.local/bin`. This is, up to the path of the virtual environment and
+the mkosi binary, equivalent to
 ```shell
-python3 -m pip install --user git+https://github.com/systemd/mkosi.git
+python -m venv mkosivenv
+mkosivenv/bin/pip install git+https://github.com/systemd/mkosi.git
+# the mkis binary is installed to mkosivenv/bin/mkosi
 ```
 
-If you want to hack on mkosi do
+If you want to help develop mkosi you can run it from your clone of this
+repository by calling the module
 ```shell
-# clone either this repository or your fork of it
-git clone https://github.com/systemd/mkosi/
-cd mkosi
-python3 -m pip install --user --no-use-pep517 --editable .
+python3 -m mkosi
 ```
-This will install mkosi in editable mode to `~/.local/bin/mkosi`, allowing you
-to use your own changes right away.
+when you are in the repository top level.
 
-For development you also need [mypy](https://github.com/python/mypy), for type
-checking, and [pytest](https://github.com/pytest-dev/pytest), to run tests.
+To use your local mkosi without being in the top level of the repository you can
+call the shim `bin/mkosi` or symlink it somewhere in your `PATH` or by making an
```

Comment:
This reads rather weird, needs rewording

Reply:
Split the sentence up some more, hopefully it's clearer now.